.travis.yml

```diff
@@ -14,6 +14,8 @@ matrix:
       dist: xenial
```

Comment:
We are moving away from Travis for tests (it's very unreliable), the tests way pay attention to are in here: https://github.com/aws/aws-toolkit-jetbrains/tree/master/buildspec

Reply:
Not sure how to install cfn-lint for the Windows tests. cfn-lint isn't available in chocolatey.

Reply:
should be able to get Python and `pip` installed on Windows and install through that